Makefile

```diff
@@ -0,0 +1,151 @@
+
+## Fill in this file to specify the project being created, and the referenced
+## Makefile-Engine will do all of the hard work for you. This handles any
+## architecture of Haiku.
+
+# The name of the binary.
+NAME = BeMobile
+
+# The type of binary, must be one of:
+#	APP:	Application
+#	DRIVER: Kernel driver
+#	SHARED:	Shared library or add-on
+#	STATIC:	Static library archive
+TYPE = APP
+
+# 	If you plan to use localization, specify the application's MIME signature.
+APP_MIME_SIG = 
+
+#	The following lines tell Pe and Eddie where the SRCS, RDEFS, and RSRCS are
+#	so that Pe and Eddie can fill them in for you.
+#%{
+# @src->@ 
+
+#	Specify the source files to use. Full paths or paths relative to the 
+#	Makefile can be included. All files, regardless of directory, will have
+#	their object files created in the common object directory. Note that this
+#	means this Makefile will not work correctly if two source files with the
+#	same name (source.c or source.cpp) are included from different directories.
+#	Also note that spaces in folder names do not work well with this Makefile.
+SRCS =  \
+         Pattern.cpp \
+	 Matcher.cpp \
+	 santa/BetterScrollView/BetterScrollView.cpp \
+	 santa/ColumnListView/CLVColumnLabelView.cpp \
+	 santa/ColumnListView/CLVEasyItem.cpp \
+	 santa/ColumnListView/CLVListItem.cpp \
+	 santa/ColumnListView/ColumnListView.cpp \
+	 santa/Cursors/Cursors.cpp \
+	 santa/NewStrings/NewStrings.cpp \
+	 santa/PrefilledBitmap/PrefilledBitmap.cpp \
+	 santa/ScrollViewCorner/ScrollViewCorner.cpp \
+	 phonelist.cpp \
+	 callregslotview.cpp \
+	 phonelistslotview.cpp \
+	 bemobile.cpp \
+	 dialabout.cpp \
+	 dialeditpbook.cpp \
+	 dialnewsms.cpp \
+	 dialcalevent.cpp \
+	 dialcalendar.cpp \
+	 gsm.cpp \
+	 main.cpp \
+	 mainwindow.cpp \
+	 workview.cpp \
+	 calview.cpp \
+	 pbbynameview.cpp \
+	 pbview.cpp \
+	 smsview.cpp \
+	 smsboxview.cpp \
+	 smsinboxview.cpp \
+	 statusview.cpp \
+	 mobileview.cpp \
+	 initwindow.cpp
+
+#	Specify the resource definition files to use. Full or relative paths can be
+#	used.
+RDEFS = Resource.rdef
+
+#	Specify the resource files to use. Full or relative paths can be used.
+#	Both RDEFS and RSRCS can be utilized in the same Makefile.
+RSRCS = 
+
+# End Pe/Eddie support.
+# @<-src@ 
+#%}
+
+#	Specify libraries to link against.
+#	There are two acceptable forms of library specifications:
+#	-	if your library follows the naming pattern of libXXX.so or libXXX.a,
+#		you can simply specify XXX for the library. (e.g. the entry for
+#		"libtracker.so" would be "tracker")
+#
+#	-	for GCC-independent linking of standard C++ libraries, you can use
+#		$(STDCPPLIBS) instead of the raw "stdc++[.r4] [supc++]" library names.
+#
+#	- 	if your library does not follow the standard library naming scheme,
+#		you need to specify the path to the library and it's name.
+#		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
+LIBS =  libbe.so libroot.so libdevice.so $(STDCPPLIBS)
```

Comment:
As it says in line 80ff, this should be just "be device $(STDCPPLIBS)". "root" is included by default.